components/ProposalStatusBadge.tsx

```diff
@@ -41,10 +44,19 @@ function getProposalStateStyle(state: ProposalState) {
 const ProposalStateBadge = ({
   proposalPk,
   proposal,
+  open,
 }: {
   proposalPk: PublicKey
   proposal: Proposal
+  open: boolean
 }) => {
+  useEffect(() => {
+    console.log('STATUS PROPOSAL', proposal.state)
+  }, [])
+
+  const [showFinalizeVoteModal, setShowFinalizeVoteModal] = useState(false)
```

Comment:
Finalise/Cancel buttons shouldn't be part of the Badge component. Let's put them in the Action panel where we have Approve/Deny/Release buttons 
<img width="464" alt="image" src="https://user-images.githubusercontent.com/5817016/146166295-e40d35ef-4299-4fa4-98b7-5e71b36b3668.png">

All the buttons (including Approve/Deny) should be displayed accordingly to the Proposal state (compare to Oyster)